src/Model/Contact.php

```diff
@@ -1372,6 +1377,7 @@ public static function getIdForURL(string $url = null, int $uid = 0, $update = n
 			$fields = [
 				'uid'       => $uid,
 				'url'       => $data['url'],
+				'baseurl'   => $data['baseurl'] ?? '',
```

Comment:
What about the related `gsid` field? The `baseurl` is used to show the Ignore Server label but it's the `gsid` we need to actually perform the action.

Reply:
In `Contact::insert` we are creating the `gsid` based on the `baseurl`.